main.py

```diff
@@ -87,16 +90,18 @@ def get_latest_tweet_id():
         return 0
 
 def main():
-    webhook_url = config['webhook_url']
-    url = create_url(config['twitter_account']) # get a url to the twitter api with the account_id inserted into it
-    params = get_params() # get the parameters to send to the API via a request 
     while True:
+        webhook_url = config['webhook_url']
+        url = create_url(config['twitter_account']) # get a url to the twitter api with the account_id inserted into it
+        params = get_params() # get the parameters to send to the API via a request 
         json_response = connect_to_endpoint(url, params) # make a request to the api endpoint at the <url> using the <params>   
         latest_tweet_id = get_latest_tweet_id()
         if int(json_response['meta']['newest_id']) > latest_tweet_id:
             tweet_urls = []
             for tweet in json_response['data']:
-                tweet_urls.append(filter_json(tweet, latest_tweet_id))
+                url = filter_json(tweet, latest_tweet_id)
+                if url:
```

Comment:
to account for `filter_json()` returning `none`, we only add the URL to the list of url's we send to discord if its not `none`